src/Analyser/NodeScopeResolver.php

```diff
@@ -1817,6 +1819,10 @@ function (MutatingScope $scope) use ($expr, $nodeCallback, $context): Expression
 			) {
 				$arrayArg = $expr->getArgs()[0]->value;
 				$arrayArgType = $scope->getType($arrayArg);
+
+				$countExpr = new FuncCall(new Name('count'), [$expr->getArgs()[0]]);
```

Comment:
atm this line depends on whether the code beeing analyzed is in a namespace or not